pytket/pytket/zx/tensor_eval.py

```diff
@@ -25,7 +25,7 @@
 except ModuleNotFoundError as err:
     err.msg = (
         'Missing package for tensor evaluation of ZX diagrams. Run "pip '
-        'install pytkex[ZX]" to install the optional dependencies and try again.'
+        'install \'pytkex[ZX]\'" to install the optional dependencies and try ' 'again.'
```

Comment:
Is the pytex intended or is this a typo?

Reply:
Typo, my bad